### PR TITLE
Create download zip file button.

### PR DESCRIPTION
**Pull request by sophiekovalevsky**
I have added the zip file button on the UI. So each time a user do click on extract, it will generate the zip button url. The url needs to be different since now you want the js approach.
I changed the jquery link since it was not working as well.

### CODE REVIEW

File: sample/index.php [hunk 1]
```diff
@@ -112,8 +113,16 @@ function isValidURL($url){
...
 </div>
 
-<script src="js/jquery.min.js"></script>
+<script src="js/jquery-2.2.0.min.js"></script>
 <script src="js/bootstrap.min.js"></script>
-
+<script>
+$( document ).ready(function() {
+  $("#form-extractor").submit(function() {
+    console.log("kiara");
```
Comment (prabhakar267) [inline sample/index.php]: Please remove this.

File: sample/index.php [hunk 2]
```diff
@@ -112,8 +113,16 @@ function isValidURL($url){
     </div>
 </div>
 
-<script src="js/jquery.min.js"></script>
+<script src="js/jquery-2.2.0.min.js"></script>
 <script src="js/bootstrap.min.js"></script>
-
+<script>
+$( document ).ready(function() {
+  $("#form-extractor").submit(function() {
```
Comment (prabhakar267) [inline sample/index.php]: This actually prevents original "Extract" functionality to fail
Please address that as well

File: sample/index.php [hunk 3]
```diff
@@ -29,10 +29,11 @@
...
             <div class="col-md-12">
-                <form class="form-horizontal form-main" method="GET">
-                      <input type="text" class="form-control" name="url" placeholder="Enter the URL from where images are to be extracted" required>
-                      <br>
-                      <button type="submit" class="btn btn-lg btn-success">Extract</button>
+                <form id="form-extractor" class="form-horizontal form-main" method="GET">
+                    <input type="text" class="form-control" name="url" placeholder="Enter the URL from where images are to be extracted" required>
+                    <br>
+                    <button type="submit" class="btn btn-lg btn-success">Extract</button>
+                    <div id="download-file"></div>
```
Comment (prabhakar267) [inline sample/index.php]: You could have the button HTML here only, which is initially hidden and after the results are loaded, button could be shown

**Comment by sophiekovalevsky** [comment]
@prabhakar267 I will be doing changes today at night. Thanks a lot for the feedback.

**Comment by sophiekovalevsky** [comment]
@prabhakar267 I have made the changes.

Instead of using jquery I have put the zip button when user exec extract button and the url is valid. No jquery approach is required anymore. 

Let me know,

**Comment by prabhakar267** [comment]
Merged to master! :smile:

**Pull request closed without merging**